source/AI.cpp

```diff
@@ -2425,10 +2425,13 @@ void AI::Attack(Ship &ship, Command &command, const Ship &target)
 
 	// Check if this ship is fast enough to keep distance from target.
 	// Have a 10% minimum to avoid ships getting in a chase loop.
+	// Ships able to run backwards can always try to move away whilst firing.
 	const bool isAbleToRun = target.MaxVelocity() * SAFETY_MULTIPLIER < ship.MaxVelocity();
```

Comment:
This check still seems to be affecting ships with the artillery personality, which I wouldn't expect to be the case. Perhaps comparing max velocities is a bad check, or maybe it's max velocity alone that's bad. Perhaps ships should check if the target is actually close to them and check if the target is actually moving. Something that's not close to you or moving means you're free to stay away from it.

Reply:
often most targets are close to each others and it's more about avoiding the action, isnt it?

Reply:
Yes, but my point is that this check doesn't achieve that well. If an artillery ship is targeting a slow ship then it will think that it's able to run away and so it will fly off and away from the action. But say it destroys its target and then retargets to a fast ship; if it's slower than its new target, it'll think it can't run away and so decide to fly into combat, even if the new target is in the middle of the action and the ship was already safe and away from it.

Reply:
we could just check for being out of weapon's range of the target if they're below our own?

Reply:
we either do this ship by ship or we can do the same as cloaking away does and run from the main action
ship by ship is prob easier to do